File: oncology-quest-api/src/utilities/regex.rs
pub(super) const USERNAME_REGEX: &str = r"^[a-zA-Z0-9\-\_\.]{1,25}$";
pub(super) const NAME_REGEX: &str = r"^[a-zA-Z\s]{1,35}$";
pub(super) const PASSWORD_REGEX: &str = r"^.{8,200}$";
pub(super) const COMMENT_REGEX: &str = r#"^[a-zA-Z0-9\s.,!?'"()-]{0,150}$"#;
pub(super) const ENTRY_TITLE_REGEX: &str = r"^[a-zA-Z0-9+\-/()\s]{1,20}$";

File: oncology_quest_mobile/lib/src/utilities/regex.dart
final RegExp usernameRegex = RegExp(r'^[a-zA-Z0-9\-_\.]{1,25}$');
final RegExp nameRegex = RegExp(r'^[a-zA-Z\s]{1,35}$');
final RegExp passwordRegex = RegExp(r'^.{8,200}$');
final RegExp commentRegex = RegExp(r'^[a-zA-Z0-9\s.,!?"()-]{0,150}$');
final RegExp entryTitleRegex = RegExp(r'^[a-zA-Z0-9+\-/()\s]{1,100}$');
final RegExp resetTokenRegex = RegExp(r'^[a-zA-Z0-9]{4}$');